doc/userguide/rules/frame-keywords.rst

```diff
@@ -0,0 +1,80 @@
+Frame Keywords
```

Comment:
think it would be good to add a block on the shorthand notation, e.g.

with `tls.pdu`, you can use it as `alert tcp ... frame:pdu`.

Reply:
TIL. Thanks. :P